src/app/homework/modals/CreateHomeworkModel.css

```diff
@@ -0,0 +1,18 @@
+
+.create-homework-modal #create-btn {
+    width: 85%;
+    height: 38px;
+    background: #91CB46;
+    color: white;
+}
+
+.create-homework-modal .schedule-item * {
+    line-height: 25px;
+    height: 25px;
+}
+
+.create-homework-modal .dot {
+    height: 10px;
+    margin-top: auto;
+    margin-bottom: auto;
+}
```

Comment:
use `scss` syntac to nest the styles
1. change the file's extension name from `.css` to `.scss`
2. Nest the styles as belows
```scss
.create-homework-modal {
    #create-btn {
         ..
    }
    .dot {
       ...
    }
}
```